just-install.json

```diff
@@ -669,6 +669,16 @@
       },
       "version": "latest"
     },
+    "slack": {
+      "installer": {
+        "kind": "as-is",
+        "options": {
+          "extension": ".exe"
+        },
+        "x86": "https://slack.com/ssb/download-win"
```

Comment:
Sorry, should have included my research. The .msi does the unexpected thing of waiting until the next time the user logs in to execute the install. So `just-install` would finish successfully, but the user would be unable to find the app. Also, the .exe is completely silent. Thoughts @lvillani ?